spec/helpers.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Helpers
```

Comment:
I think it's a good idea to create a `spec/support` directory, and maybe we can put this file there

Also recommend to rename this file, maybe `ResponseHelpers`? This way we don't keep adding methods to this file and just create new ones for different purposes